### ISSUE TITLE
resource icon vanish on restart -- save file

### ISSUE THREAD

**Issue by aviator99** [civfanatics/CQUI_Community-Edition#283]
as per request on steam discussion, not sure if anyone else has provided but couldn't immediately see one. I can provide more if required, anything else to help.
[JOÃO III 167 110 BC - Copy.zip](https://github.com/civfanatics/CQUI_Community-Edition/files/6245804/JOAO.III.167.110.BC.-.Copy.zip)

**Comment by the-m4a** [comment]
thank you!  checking it out now

**Comment by the-m4a** [comment]
Started game, loaded your save, and the resource icons were all there. :/

![image](https://user-images.githubusercontent.com/8787640/113469599-97f28c00-9403-11eb-91bd-773b2b670b3b.png)

**Comment by the-m4a** [comment]
Ok I found an event later than LoadGameViewStateDone, maybe that will do the trick.  I'll upload it to the Steam workshop and create a PR here.  

Attached is a save where I placed every kind of resource on the map, just to verify everything would load and show up.  Had no issue.

![image](https://user-images.githubusercontent.com/8787640/113470403-3386fb00-940a-11eb-842e-929b332b4c93.png)

[AMBIORIX 60 640 BC.zip](https://github.com/civfanatics/CQUI_Community-Edition/files/6252047/AMBIORIX.60.640.BC.zip)
